.gitignore

```diff
@@ -1 +1,2 @@
+.idea
```

Comment:
Please use `.git/info/exclude` for editor-specific hidden directories and files.